assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py

```diff
@@ -1285,11 +1291,20 @@ def _parse_responses(
             print(f"Failed decoding examples: {sample_examples}")
             num_failed += 1
 
+    # if the number of expected samples does not match the output samples
+    # there was an issue getting the response and we cannot accurately map
+    # the indices with the input data
+    index_mapping = job.prompt_data.input_idx
```

Comment:
Aren't those just 1-20, for example, for every batch? I might be misremembering.

Reply:
I've only tested on a dataset with 14 but there's two batches and I get 0-9 in one batch and 10-13 in the next. I can test with a larger dataset to confirm but seems like it's not

Reply:
just tested with a dataset that has 55 entries and there was an input_idx for each entry even though the batches were smaller than that